quarkus-extension/simple-rest-example/README.md

```diff
@@ -0,0 +1,21 @@
+# Camunda Platform Runtime Quarkus Example
+
+This project uses Quarkus, the Supersonic Subatomic Java Framework.
+
+If you want to learn more about Quarkus, please visit its website: https://quarkus.io/ .
+
```

Comment:
I think we should add an introduction on what the example does. Something similar to the REAME here https://github.com/camunda/camunda-bpm-examples/tree/master/spring-boot-starter/example-simple maybe?

Reply:
👍